server/src/main/java/edu/jhu/server/data/ILocation.java

```diff
@@ -0,0 +1,5 @@
+package edu.jhu.server.data;
+
+public interface ILocation {
+  ILocation[] getConnectedLocations();
```

Comment:
i figured Board would need some way to get connected locations of certain locations (so it can send the list of valid moves to client for instance). another possible solution is to just have Board know whats connected to what. that might cleanup the Room/Hallway stuff.